x-pack/plugins/security_solution/common/field_maps/8.13.0/alerts.ts

```diff
@@ -6,10 +6,35 @@
  */
 
 import { alertsFieldMap840 } from '../8.4.0';
-import { ALERT_HOST_CRITICALITY, ALERT_USER_CRITICALITY } from '../field_names';
+import {
+  ALERT_HOST_CRITICALITY,
+  ALERT_USER_CRITICALITY,
+  LEGACY_ALERT_HOST_CRITICALITY,
+  LEGACY_ALERT_USER_CRITICALITY,
+  ALERT_HOST_RISK_SCORE_CALCULATED_LEVEL,
+  ALERT_HOST_RISK_SCORE_CALCULATED_SCORE_NORM,
+  ALERT_USER_RISK_SCORE_CALCULATED_LEVEL,
+  ALERT_USER_RISK_SCORE_CALCULATED_SCORE_NORM,
+} from '../field_names';
 
 export const alertsFieldMap8130 = {
   ...alertsFieldMap840,
+  /**
+   * @deprecated Use ALERT_HOST_CRITICALITY instead.
```

Comment:
I wasn't able to see these JSDoc being used anywhere, either on the broader `alertsFieldMap8130` type, nor on pulling the value out. Since the key itself is deprecated, perhaps that's good enough?